DiceKeys/Views/Scanning/CameraFrameCountView.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CameraFrameCountView.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 26/01/21.
+//
+
+import SwiftUI
+
+/// CameraFrameCountModel
```

Comment:
A comment with the name that appears below is of no value.  Perhaps...

```
// This model tracks the frame count in an object that ScanDiceKey needn't
// observe, so that only the child that needs to render the frame count
// (CameraFrameCountView, below) refreshes on every frame.
```